### DISCUSSION TITLE
Search inconsistency + Sort by not workinh

### DISCUSSION THREAD

**Discussion by lpoirothattermann** [community/community#113801] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

- Every refresh you get different Repository count (e.g "More than 833k results", "More than 830k results", "More than 814k results" ...): https://github.com/search?q=language%3AGo&type=Repositories
- Sort by most stars not working https://github.com/search?q=language%3AGo&type=Repositories&s=stars&o=desc to make it work need to indicate minimum stars in the query https://github.com/search?q=stars%3A%3E1000+language%3AGo&type=Repositories&ref=advsearch&l=Go&l=&s=stars&o=desc

**Comment by hebuildapps** [comment]
Heyy @lpoirothattermann 
I have mentioned few resources, for help you surely can refer those : 
https://github.com/PyGithub/PyGithub/issues/2812
This one was the best answer, which I found .. 😉
